config.json

```diff
@@ -199,7 +199,12 @@
       "core": true,
       "difficulty": 4,
       "slug": "matrix",
-      "topics": null,
+      "topics": [
+	"loops",
+	"arrays",
+	"pattern_matching",
+	"integers"
```

Comment:
Great suggestions! :) I would also add `strings` since it uses strings, and `type_conversion` since you have to convert a `string` to an `int`. Do you agree? :)

Also all these topics should be indented two spaces in from `topics`. And the closing square bracket should be indented in line with `topics` and `unlocked_by`. Does that make sense? :)